unused/Battle_of_Jingxing.md

```diff
@@ -0,0 +1,8 @@
+The Battle of Jingxing was fought in 205 BC between forces of the Han kingdom and those of the Zhao. The Han numbered 30,000 and were commanded by Han Xin. The Zhao led by commander Chen Xu were, with their 200,000 men, not only numerically superior but also, since the Han army consisted entirely of peasant militia, more experienced. This was because Han Xin, the Han commander, had been obligated to send his best man to his warlord.
```

Comment:
Whose warlord?

Reply:
Does it say who the warlord was?

Reply:
Sorry, I apprarantly overlooked this comment. It is Liu Bang.